caravel/controllers/forms.py

```diff
@@ -33,11 +33,11 @@ class ListingForm(flask_wtf.Form):
     
 
 class NewListingForm(ListingForm):
-    principal = PrincipalField("Seller",
-        validators=[DataRequired(), OnlyValid()])
+    principal = PrincipalField("Seller", validators=[DataRequired()])
```

Comment:
Does this prevent somebody from using their Uchicago email to make a new listing?
i.e., do you want another ```MatchesPrincipal()``` that prevents CnetID's from posting listings this way?